src/main/java/uk/gov/ida/verifyserviceprovider/services/EidasAssertionTranslator.java

```diff
@@ -19,22 +14,10 @@
 import uk.gov.ida.verifyserviceprovider.validators.SubjectValidator;
 
 import java.util.List;
-import java.util.Optional;
 
 import static java.util.Collections.singletonList;
-import static uk.gov.ida.verifyserviceprovider.dto.NonMatchingScenario.IDENTITY_VERIFIED;
-
-public class EidasAssertionTranslator extends IdentityAssertionTranslator {
-
-    private final InstantValidator instantValidator;
-    private final ConditionsValidator conditionsValidator;
-    private final LevelOfAssuranceValidator levelOfAssuranceValidator;
-    private final EidasMetadataResolverRepository metadataResolverRepository;
-    private final SignatureValidatorFactory signatureValidatorFactory;
-    private final List<String> acceptableHubConnectorEntityIds;
-    private UserIdHashFactory userIdHashFactory;
-    private final AuthnContextFactory authnContextFactory = new AuthnContextFactory();
 
+public class EidasAssertionTranslator extends BaseEidasAssertionTranslator {
```

Comment:
... and this is so much clearer about what it does as a result of the abstract base class.